app/src/main/java/org/buffer/android/components/RoundedButton.kt

```diff
@@ -16,7 +16,7 @@ class RoundedButton @JvmOverloads constructor(
 
     init {
         setBackground(attrs)
-        gravity = CENTER_HORIZONTAL
+        gravity = CENTER
```

Comment:
This is to fix the alignment issue we were seeing with like and close where there's a compound drawable added